doc: clarify rdbms configuration (#9329)
camunda/camunda-docs: docs/self-managed/concepts/databases/relational-db/configuration.md

## Enable RDBMS as secondary storage

> Setting `camunda.data.secondary-storage.type: rdbms` activates the full RDBMS backend in a single step: it automatically enables the RDBMS exporter (which streams workflow data to the database) and configures the application layer (Operate, Tasklist, Identity, REST API) to use RDBMS for secondary storage.

Reviewer: ```suggestion
Set the `camunda.data.secondary-storage.type` property to `rdbms` to activate the full RDBMS backend in a single step. This automatically enables the RDBMS exporter, which streams workflow data to the database, and configures the application layer (Operate, Tasklist, Identity, REST API) to use RDBMS for secondary storage.
```
